### DISCUSSION TITLE
use .umap file with pre-built carla without recompiling

### DISCUSSION THREAD

**Discussion by Aliikram9716** [carla-simulator/carla#8248] [Q&A]
hi, i am working on a project for which i need to use custom maps and for stability purpose i have to make a package of carla and use that i.e use prebuilt carla, i also need the environment information along with the road network information, is there a way to load the .umap file onto prebuilt carla without recompiling carla?

**Comment by Jinviz** [comment]
After the initial build (make launch) is complete run carlaUE4.uproject in the unreal/content folder, you can run carla projects in a built state. If that helped, please complete that post :)
